### PR TITLE
refactor(capture-sdk): Delete example-app-shared-code

### PR DESCRIPTION
**Pull request by mahdiabolfazli**
Moved the example-app-shared-code files to screen-api-example-app. The package was not needed anymore because we had a single example app (before there was another example app in the capture-sdk).

### CODE REVIEW

File: capture-sdk/screen-api-example-app/src/main/java/net/gini/android/capture/screen/BaseExampleApp.java
Comment (a-szotyori) [inline capture-sdk/screen-api-example-app/src/main/java/net/gini/android/capture/screen/BaseExampleApp.java:1]: [Suggestion] What do you think of moving all the code from `BaseExampleApp` into `ScreenApiExampleApp`?

File: settings.gradle.kts [hunk 1]
```diff
@@ -26,7 +26,7 @@ include("health-sdk:example-app")
 
 include("capture-sdk:sdk")
 include("capture-sdk:default-network")
-include("capture-sdk:example-app-shared-code")
+//include("capture-sdk:example-app-shared-code")
```
Comment (a-szotyori) [inline settings.gradle.kts:29]: [Request] You can delete commented out lines.

**Review by a-szotyori — CHANGES_REQUESTED** [review]
Looks good. I just left one request and one suggestion.

File: capture-sdk/screen-api-example-app/src/main/java/net/gini/android/capture/screen/BaseExampleApp.java
Comment (mahdiabolfazli) [reply capture-sdk/screen-api-example-app/src/main/java/net/gini/android/capture/screen/BaseExampleApp.java:1]: I did not do any modifications here, I will definitely do it when I am refactoring from Java to Kotlin and restructuring the packages. Is it okay?

**Pull request merged**